client/src/components/reader/Annotation/Popup/Annotate.js

```diff
@@ -0,0 +1,57 @@
+import React, { PureComponent, PropTypes } from 'react';
+import HigherOrder from 'containers/global/HigherOrder';
+
+export default class AnnotationPopupAnnotate extends PureComponent {
+
+  static displayName = "Annotation.Popup.Annotate";
+
+  static propTypes = {
+    secondary: PropTypes.string,
+    attachResource: PropTypes.func,
+    highlight: PropTypes.func,
+    annotate: PropTypes.func,
+    bookmark: PropTypes.func,
+    showShare: PropTypes.func,
+    pageClass: PropTypes.string,
```

Comment:
Passing in a class from the parent to the child in this case has a hint of bad code smell to me. It might be fine, but to my eyes exposing a class like this to the parent feels like a leaky abstraction. Why would the annotate and share panels ever have a different class than the one they've been given here? If there is a valid case for the parent to set their appearance, shouldn't it be by way of a directive rather than by passing the child a class, which is generally an implementation detail?